app/build.gradle

```diff
@@ -77,7 +77,6 @@ android {
             signingConfig signingConfigs.debug
             applicationIdSuffix '.debug'
             versionNameSuffix "-debug"
-            shrinkResources true
```

Comment:
Sorry for the delay in my update. The changes in the size and the number of methods are as below:

Size: 4.7MB → 5.6MB in Raw Size
![droidkaigi2016_apk_vs_app-production-debug_apk_and_droidkaigi2016_apk_-_fril-android_-____git_github_com_fablic_fril-android__-_android_studio_2_2_preview_7](https://cloud.githubusercontent.com/assets/6446183/17514478/cb881772-5e6d-11e6-89ec-b85f950b60a6.png)

Number of Methods:
**Before**
![droidkaigi2016_apk_-_fril-android_-____git_github_com_fablic_fril-android__-_android_studio_2_2_preview_7](https://cloud.githubusercontent.com/assets/6446183/17514534/14c87ce2-5e6e-11e6-88d1-e76498437cfa.png)

**After**
![app-production-debug_apk_-_fril-android_-____git_github_com_fablic_fril-android__-_android_studio_2_2_preview_7](https://cloud.githubusercontent.com/assets/6446183/17514544/1faaf860-5e6e-11e6-984d-17050698e369.png)

Thanks!